src/basics.md

```diff
@@ -1266,6 +1267,37 @@ fn main() {
 }
 ```
 
+[ex-convert-datetime-timestamp]: #ex-convert-datetime-timestamp
+<a name="ex-convert-datetime-timestamp"></a>
+## Converting date to unix timestamp and vice versa
```

Comment:
- Please use imperative form instead of continuous one in the example title.
s/Converting/Convert/
- UNIX is all caps